dex/src/main/scala/com/wavesplatform/dex/AddressActor.scala

```diff
@@ -301,22 +309,26 @@ class AddressActor(
 }
 
 object AddressActor {
+
   private val ExpirationThreshold = 50.millis
 
   private type Resp = api.MatcherResponse
 
-  private def activeStatus(lo: LimitOrder): OrderStatus =
-    if (lo.amount == lo.order.amount) OrderStatus.Accepted else OrderStatus.PartiallyFilled(lo.order.amount - lo.amount)
+  private def activeStatus(ao: AcceptedOrder): OrderStatus =
+    if (ao.amount == ao.order.amount) OrderStatus.Accepted else OrderStatus.PartiallyFilled(ao.order.amount - ao.amount)
 
   sealed trait Command
 
-  case class GetOrderStatus(orderId: ByteStr)                             extends Command
-  case class GetOrders(assetPair: Option[AssetPair], onlyActive: Boolean) extends Command
-  case class GetTradableBalance(assetPair: AssetPair)                     extends Command
-  case object GetReservedBalance                                          extends Command
-  case class PlaceOrder(order: Order) extends Command {
-    override lazy val toString = s"PlaceOrder(${order.id()},${order.sender},${order.assetPair},${order.orderType},${order.price},${order.amount})"
+  case class GetOrderStatus(orderId: ByteStr)                                     extends Command
+  case class GetOrdersStatuses(assetPair: Option[AssetPair], onlyActive: Boolean) extends Command
+  case class GetTradableBalance(assetPair: AssetPair)                             extends Command
+  case object GetReservedBalance                                                  extends Command
+
+  case class PlaceOrder(order: Order, isMarketOrder: Boolean = false) extends Command {
+    override lazy val toString = s"${if (isMarketOrder) "PlaceMarketOrder" else "PlaceOrder"}(${order
```

Comment:
May be something like this? 
```
s"PlaceOrder(${if (isMarket) "market," else ""}
```
It's easy to forget that PlaceOrder is actually "PlaceOrder" and "PlaceMarketOrder".

Reply:
Do you suggest to split that command into 2 (`PlaceLimitOrder` and `PlaceMarketOrder`)?